examples/apps/Dockerfile.centrifugo

```diff
@@ -0,0 +1,12 @@
+FROM golang:1.16
+
+RUN git clone https://github.com/centrifugal/centrifugo.git $GOPATH/src/centrifugo
+WORKDIR $GOPATH/src/centrifugo
+
+COPY main/centrifugo main.go
+RUN GO111MODULE=on go mod tidy
+RUN GO111MODULE=on go build
+RUN chmod +x centrifugo
+RUN ./centrifugo genconfig
+
+ENTRYPOINT ["./centrifugo"]
```

Comment:
Please make sure your text editor adds new lines at the end of each file. More info here: https://pyroscope.io/docs/style-guide#other-considerations

Reply:
ok

Reply:
done